code/modules/telesci/bscrystal.dm

```diff
@@ -15,6 +15,21 @@
 	usesound = 'sound/items/deconstruct.ogg'
 	dynamic_icon_state = TRUE
 
+/obj/item/stack/ore/bluespace_crystal/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>You can crush this to randomly teleport a short distance.</span>"
+	. += "<span class='notice'>If you throw this at someone, they'll be randomly teleported a short distance away.</span>"
+
+/obj/item/stack/ore/bluespace_crystal/examine_more(mob/user)
+	..()
+	. = list()
```

Comment:
The normal bluespace crystal doesn't need to override the entire `examine_more()`, right?

Reply:
As it turns out, no.